chat-app-express/app.js

```diff
@@ -4,31 +4,33 @@ var path = require('path');
 var cookieParser = require('cookie-parser');
 var logger = require('morgan');
 var indexRouter = require('./src/routes/index');
-var usersRouter = require('./src/routes/users');
+var usersRouter = require('./src/routes/user-routes');
 var mongoose = require('mongoose');
 
 var app = express();
 
-require('dotenv').config()
+require('dotenv').config();
 
-const PORT = process.env.PORT;
+const PORT = process.env.PORT || 3000; // Set a default port if PORT environment variable is not set
 const http = require('http');
 const socket = require("socket.io");
 const server = http.createServer(app);
 const io = socket(server);
-// view engine setup
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'jade');
 
 app.use(logger('dev'));
 app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
 app.use(cookieParser());
-app.use(express.static(path.join(__dirname, 'public')));
+app.use(express.static('src/public'));
 
 app.use('/api/', indexRouter);
 app.use('/api/users', usersRouter);
 
+// DB connection
+mongoose.connect(process.env.DBLINK)
```

Comment:
make a separate file for mongodb connect (connect.js) export connection function and use it here

Reply:
Done